crates/rolldown/tests/esbuild/default/es6_from_common_js/artifacts.snap

```diff
@@ -23,11 +25,12 @@ var require_bar = __commonJS({ "bar.js"(exports) {
 		return "bar";
 	};
 } });
+var import_bar = __toESM(require_bar());
 
 //#endregion
 //#region entry.js
-var import_foo = __toESM(require_foo());
-var import_bar = __toESM(require_bar());
+require_foo();
```

Comment:
These calls are no longer necessary and should be removed

Reply:
done